zebra-network/src/protocol/internal.rs

```diff
@@ -12,15 +17,18 @@ use super::types::Nonce;
 
 /// A network request, represented in internal format.
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[cfg_attr(test, derive(Arbitrary))]
 pub enum Request {
     /// Requests additional peers from the server.
     GetPeers,
     /// Advertises peers to the remote server.
+    #[cfg_attr(test, proptest(skip))]
```

Comment:
Are these skipped because `MetaAddr`s are not `Arbitrary`?

Reply:
Correct and adding `Arbitrary` requires further `Arbitrary`s all the way down.

Reply:
👍  Since the proptest parts of this PR just do scaffolding and don't yet test actual properties, how would you feel about pulling them out of this PR and moving them to a PR that did something like test round-trip serialization of `Message`s? It is a bigger chunk because it requires propagating `Arbitrary`, but it seems like it has less risk of leaving stub code in the tree.

Reply:
Definitely was thinking about doing this, will pull out to its own branch.